.github/scripts/run-local-buildbitstreams.py

```diff
@@ -223,12 +222,16 @@ def replace_in_hwdb(hwdb_entry_name: str, link: str) -> None:
             # priority == roughly the more powerful and available
             # ipaddr, buildtool:version, use unique build dir, unique build dir path, priority (0 is highest)(unused by code but used to track which machine has most resources)
             hosts = [
+                (          "as4",  "vivado:2022.1", False, "", 0),
+                ("buildbot1@as4",  "vivado:2022.1",  True, "/scratch/buildbot1/FIRESIM_BUILD_DIR", 0),
+                ("buildbot2@as4",  "vivado:2022.1",  True, "/scratch/buildbot2/FIRESIM_BUILD_DIR", 0),
+                (    "localhost",   "vitis:2022.1", False, "", 0),
                 ("buildbot1@a17",   "vitis:2022.1",  True, "/scratch/buildbot1/FIRESIM_BUILD_DIR", 0),
-                (         "harp",   "vitis:2022.1", False, "", 2),
                 ("buildbot2@a17",   "vitis:2021.1",  True, "/scratch/buildbot2/FIRESIM_BUILD_DIR", 0),
                 ("buildbot3@a17",   "vitis:2021.1",  True, "/scratch/buildbot3/FIRESIM_BUILD_DIR", 0),
                 ("buildbot4@a17",   "vitis:2021.1",  True, "/scratch/buildbot4/FIRESIM_BUILD_DIR", 0),
                 (     "firesim1",   "vitis:2021.1", False, "", 1),
+                (         "harp",   "vitis:2022.1", False, "", 2),
```

Comment:
I thought harp was removed?

Reply:
It currently still exists as a build machine (with no FPGA). We can easily remove this later if the machine gets removed.